README.md

```diff
@@ -1,54 +1,68 @@
 
-[Temporal](https://github.com/temporalio/temporal) is a distributed, scalable, durable, and highly available orchestration engine used to execute asynchronous long-running business logic in a scalable and resilient way.
+The Temporal Go SDK provides a framework for Temporal Application development in the Go language.
+The SDK contains the following tools:
 
-"Temporal Go SDK" is the framework for authoring workflows and activities using Go language.
+- A Temporal Client to communicate with a [Temporal Cluster](https://docs.temporal.io/docs/content/what-is-a-temporal-cluster)
+- APIs to use within your Workflows
+- APIs to create and manage [Workers](https://docs.temporal.io/docs/content/what-is-a-worker)
 
 ## How to use
 
-Clone this repo into the preferred location.
+Add the [Temporal Go SDK](https://github.com/temporalio/sdk-go) to your project:
 
 ```bash
-git clone git@github.com:temporalio/sdk-go.git
+go get -u go.temporal.io/sdk@latest
 ```
 
-or
+Or clone the Go SDK repo to your preferred location:
 
 ```bash
-go get -u go.temporal.io/sdk
-```
-
-See [samples](https://github.com/temporalio/samples-go) to get started.
-
-Documentation is available [here](https://docs.temporal.io). 
-You can also find the API documentation [here](https://pkg.go.dev/go.temporal.io/sdk).
-
-## Using a custom logger
-
-Although the Go SDK does not support most third-party logging solutions natively, [our friends at Banzai Cloud](https://github.com/sagikazarmark) built the [adapter package logur](https://github.com/logur/logur) which makes it possible to use third party loggers with minimal overhead. Here is an example of using logur to support [Logrus](https://github.com/sirupsen/logrus):
-
-```go
-package main
-import (
-	"github.com/sirupsen/logrus"
-	logrusadapter "logur.dev/adapter/logrus"
-	"logur.dev/logur"
-)
-
-func main() {
-	// feed this logger into Temporal
-	logger := logur.LoggerToKV(logrusadapter.New(logrus.New()))
-}
+git clone git@github.com:temporalio/sdk-go.git
 ```
-
-Most of the popular logging solutions have existing adapters in logur. If you're curious about which adapters are available, here is a helpful [link](https://github.com/logur?q=adapter-).
+**Are there executable code samples?**
+
+You can find a complete list of executable code samples in the [samples library](https://docs.temporal.io/docs/samples-library/#go), which includes Temporal Go SDK code samples from the [temporalio/samples-go](https://github.com/temporalio/samples-go) repo.
+Additionally, each of the Go SDK Tutorials in the [Temporal docs site](https://docs.temporal.io/docs/go) is backed by a fully executable template application.
+
+**Where is the Go SDK technical reference?**
+
+The [Temporal Go SDK API reference](https://pkg.go.dev/go.temporal.io/sdk) is published on [pkg.go.dev](https://pkg.go.dev/go.temporal.io/sdk)
+
+## How-to guides
```

Comment:
I don't love that we're putting this list here.
Let's keep it in the docs (for now at least).

Reply:
So you would prefer a single link?

Reply:
SGTM